makefiles/tools/edbg.inc.mk

```diff
@@ -3,6 +3,14 @@ EDBG ?= $(RIOT_EDBG)
 FLASHER ?= $(EDBG)
 OFLAGS ?= -O binary
 HEXFILE = $(ELFFILE:.elf=.bin)
```

Comment:
Haven't this already been fixed in the previous PR? Or it was only done for OpenOCD?

Reply:
oops, this should be IMAGE_FILE

Reply:
hmm, actually, it's still HEXFILE because that is the target of the OBJCOPY make rule

Reply:
I see, it's just that it looks wrong and needs to be fixed. I wouldn't mind to fix it in this PR but if you prefer it can be a following PR.

Reply:
I would prefer to keep this PR focused on separating the openocd configuration from the board Makefiles and do any other improvements in separate pulls.